composer.json

```diff
@@ -22,8 +22,8 @@
     "require-dev": {
         "phpro/grumphp": "^0.17.1 || ^0.19.0",
         "phpstan/phpstan": "^0.11.15 || ^0.12.0",
-        "phpunit/phpunit": ">=7.5",
-        "squizlabs/php_codesniffer": "^3.0",
+        "phpunit/phpunit": ">=9.0",
+        "squizlabs/php_codesniffer": "^3.5",
```

Comment:
The PHP_CodeSniffer `^3.0` version cannot be for `php-7.4` version.

It should use `^3.5` version instead.